index.d.ts

```diff
@@ -113,17 +143,38 @@ declare module 'plaid' {
     number_of_income_streams: number;
   }
 
+  interface Identity {
+    addresses: Array<Address>;
+    emails: Array<Email>;
+    names: Array<string>;
+    phone_numbers: Array<PhoneNumber>;
+  }
+
+  interface AccountIdentity {
```

Comment:
do we think these are going to diverge in the future? (if so, smart to separate them out now).

Reply:
This reflects what's in `apischemas/identity.go` ([source](https://github.plaid.com/plaid/go/blob/cf83209a57f77c64599d35adc6a032e40eba7272/lib/apischemas/identity.go#L20)).